src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/eøs/utenlandskperiodebeløp/UtenlandskPeriodebeløpServiceTest.kt

```diff
@@ -0,0 +1,135 @@
+package no.nav.familie.ks.sak.kjerne.eøs.utenlandskperiodebeløp
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.familie.ba.sak.kjerne.eøs.utenlandskperiodebeløp.jan
+import no.nav.familie.ba.sak.kjerne.tidslinje.util.KompetanseBuilder
+import no.nav.familie.ks.sak.common.util.førsteDagIInneværendeMåned
+import no.nav.familie.ks.sak.data.tilfeldigPerson
+import no.nav.familie.ks.sak.kjerne.eøs.differanseberegning.domene.Intervall
+import no.nav.familie.ks.sak.kjerne.eøs.felles.domene.EøsSkjemaRepository
+import no.nav.familie.ks.sak.kjerne.eøs.kompetanse.domene.KompetanseRepository
+import no.nav.familie.ks.sak.kjerne.eøs.kompetanse.mockEøsSkjemaRepository
+import no.nav.familie.ks.sak.kjerne.eøs.utenlandskperiodebeløp.domene.UtenlandskPeriodebeløp
+import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.domene.PersonType
+import no.nav.familie.ks.sak.kjerne.util.UtenlandskPeriodebeløpBuilder
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+internal class UtenlandskPeriodebeløpServiceTest {
+    val utenlandskPeriodebeløpRepository: EøsSkjemaRepository<UtenlandskPeriodebeløp> =
+        mockEøsSkjemaRepository()
+    val kompetanseRepository: KompetanseRepository = mockk()
+
+    val utenlandskPeriodebeløpService =
+        UtenlandskPeriodebeløpService(
+            utenlandskPeriodebeløpRepository,
+            emptyList(),
+        )
+
+    val tilpassUtenlandskePeriodebeløpTilKompetanserService =
+        TilpassUtenlandskePeriodebeløpTilKompetanserService(
+            utenlandskPeriodebeløpRepository,
+            emptyList(),
+            kompetanseRepository,
+        )
+
+    @BeforeEach
+    fun init() {
+        utenlandskPeriodebeløpRepository.deleteAll()
+    }
```

Comment:
Gitt at det er en enhetstest så trenger vi ikke denne siden staten ikke går over til neste test

Reply:
Testen bruker en slags Fake, så clear rydder opp i faken

Reply:
Ser ut som den kaller deleteAll som er en repostiory metode og ikke mockk clear som jeg er vant med 🤔 
Fungerte å kjøre testene uten kodesnutten også, så var litt merkelig isåfall

Reply:
Ah fett, ser det nå! 
I MockEøsSkjemaRepository 🤩

Reply:
repo er MockEØsSkjemaRepository hvor deletAll gjør skjema.clear() i den minnebaserte skjemaet. Litt merkelig blanding av mockk og minnebasert. Mulig at den alltid vil kjøre ok uten, men er jo en fare for at den går i bena på hverandre hvis man kjører i flere testene i parallell